packages/vue-components/src/PageNavButton.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <overlay
+    v-if="showPageNav"
+    type="pageNav"
+    fragment="page-nav"
+    :src="src"
+    :has-identifier="hasIdentifier"
+  >
+    <span slot="navMenuIcon" :class="['glyphicon', 'toggle-page-nav-button']"></span>
+  </overlay>
+</template>
+
+<script>
+import $ from './utils/NodeList';
+
+export default {
+  data() {
+    return {
+      hasPageNav: false,
+      hasIdentifier: false,
+      show: false,
+      src: '',
+    };
+  },
+  provide() {
+    return {
+      getNavMenuContent: this.getPageNavContent,
```

Comment:
sorry, commented on the wrong line 🙊

Reply:
Thanks for the quick clarification 👍. Updated :)